securedrop-client/debian/postinst

```diff
@@ -22,6 +22,17 @@ case "$1" in
     configure)
 
     update-desktop-database /usr/share/applications
+    # If the profile was disabled enabled it.
+    if [ -e "/etc/apparmor.d/disable/usr.bin.securedrop-client" ]; then
+        rm /etc/apparmor.d/disable/usr.bin.securedrop-client
+    fi
+
+    # if the deb package is providing a new version of the apparmor profile, squash the old one
```

Comment:
Why is this explicit handling necessary?

Reply:
Just tested locally, running aa-enforce will reliably remove the symlink from `/etc/apparmor.d/disabled/` . I will remove these lines.